### PR TITLE
maven & hello catalog endpoint init commit

### PR DESCRIPTION
**Pull request by pjgg**

### CODE REVIEW

File: rest-api/pom.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        </dependency>
+
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-jsonb</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.jboss.logmanager</groupId>
+            <artifactId>log4j2-jboss-logmanager</artifactId>
```
Comment (Sgitario) [inline rest-api/pom.xml]: What is this dependency for?

File: rest-api/src/main/java/io/quarkus/qe/HelloCatalog.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import javax.ws.rs.core.MediaType;
+
+@Path("/catalog")
+public class HelloCatalog {
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    @Path("/")
+    public String getRandomJoke() {
+        return "hello Catalog";
```
Comment (Sgitario) [inline rest-api/src/main/java/io/quarkus/qe/HelloCatalog.java:15]: I can't find the joke! I guess it's a copy/paste leftover, can you rename the method?

**Review by Sgitario — COMMENTED** [review]
Lgtm. Few comments and also to remove the .gitkeep files (and add them into the .gitignore file).

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+    <modules>
+        <module>rest-api</module>
+        <module>enrichers</module>
+    </modules>
+
+    <properties>
+        <compiler-plugin.version>3.8.1</compiler-plugin.version>
+        <maven.compiler.parameters>true</maven.compiler.parameters>
+        <maven.compiler.source>1.8</maven.compiler.source>
```
Comment (jsmrcka) [inline pom.xml]: Any reason for not using 11 (both source and target)?

**Review by Sgitario — APPROVED** [review]

**Pull request merged**